### PR TITLE
tensorboard_dir

### PR DESCRIPTION
**Pull request by Ilya-Krylov**
Satisfied feature request about saving tensorboard logs in specified folder

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run tests

File: tools/train.py [hunk 1]
```diff
@@ -154,6 +156,13 @@ def main():
     cfg_samples_per_gpu = cfg.data.samples_per_gpu
     if args.update_config is not None:
         cfg.merge_from_dict(args.update_config)
+    if args.tensorboard_dir is not None:
+        hooks = [hook for hook in cfg.log_config.hooks if hook.type == 'TensorboardLoggerHook']
+        if hooks:
+            hooks[0].log_dir = args.tensorboard_dir
+        else:
+            logging.warning('Failed to find TensorboardLoggerHook')
```
Comment (druzhkov-paul) [inline tools/train.py]: Is it possible to use the logger that is defined and initialized at line 203?

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

File: tools/train.py (hunk 1 above)
Comment (Ilya-Krylov) [reply tools/train.py]: done

**Review by druzhkov-paul — APPROVED** [review]

**Pull request merged**